client/ipa-getkeytab.c

```diff
@@ -906,6 +1085,41 @@ int main(int argc, const char *argv[])
         exit(2);
     }
 
+    if (server && (strcmp(server, "_srv_") == 0)) {
```

Comment:
Shouldn't we use `strncasecmp()` here and not just `_srv_` prefix? It would be a bit more friendly to a user...

Reply:
Sure, that's a good idea. Fixed.